src/core/accessibility.js

```diff
@@ -1,3 +1,66 @@
+import { isMac } from '../helpers/device';
```

Comment:
Do you think we can move this entire file to `helpers`? I think the `core` folder mostly just contains WebViewer APIs.
Would like to hear your thoughts. We can just leave it as it is.

Reply:
Yep moved, wasn't sure about the purposes of the folders